core/src/avm2/globals/README.md

```diff
@@ -63,6 +63,28 @@ class is loaded.
 
 See `flash/events/Event.as` for an example
 
+## API Versioning
+
+Ruffle supports Flash's Api versioning, which hides newer playerglobal definitions
+(including methods/properties) from SWFs compiled with older API versions.
+For example, see `Event.WORKER_STATE`
+
+To add versioning to an API:
+
+1. Determine the first version where it was added. This can be seen in the Flash Documentation (e.g. "Runtime Versions: Flash Player 11.4, AIR 3.4")
+2. Convert the Flash Player version to an SWF version number using [this chart](http://www.adobe.com/devnet/articles/flashplayer-air-feature-list.html)
+2. Determine the corresponding asc.jar version code for the SWF version. This can be found in avmplus in https://github.com/adobe/avmplus/blob/master/core/api-versions.as
```

Comment:
It might be useful to note that this chart can also version based on FP version

Reply:
I'm not really sure what you mean

Reply:
https://github.com/adobe/avmplus/blob/master/core/api-versions.as also has versioning based on FP minor version
(e.g. `CONFIG const FP_10_1 = 667;`)

Reply:
That's only for a few very old versions